testutil/src/main/java/tech/pegasys/ethsigner/testutil/EthSignerConfig.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2019 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.ethsigner.testutil;
+
+import tech.pegasys.ethsigner.core.Config;
+import tech.pegasys.ethsigner.core.signing.ChainIdProvider;
+
+import java.net.InetAddress;
+import java.nio.file.Path;
+import java.time.Duration;
+
+import org.apache.logging.log4j.Level;
+
+public class EthSignerConfig implements Config {
```

Comment:
These EthSigner test util classes can be moved into acceptance tests as will only be used in the acceptance tests